src-admin/src/Tabs/BridgesAndDevices.tsx

```diff
@@ -119,7 +119,8 @@ class BridgesAndDevices<TProps extends BridgesAndDevicesProps, TState extends Br
     static getVendorIcon(vendorId: number, themeType: ThemeType): React.JSX.Element | null {
         const vendor = VENDOR_IDS[vendorId];
 
-        if (vendor === 'Amazon Lab126') {
+        if (vendorId === 0x1217) {
```

Comment:
do we maybe want to have a map like structure to decode this to its real meaning?